internal/plugins/helm/v1/init.go

```diff
@@ -144,7 +144,10 @@ func (p *initPlugin) Run() error {
 		return err
 	}
 
-	// Run the scorecard "phase 2" plugin.
+	// Run SDK "phase 2" plugins.
+	if err := manifests.RunInit(p.config); err != nil {
+		return err
+	}
```

Comment:
IMO it needs to be in postScaffoldActions. 
Also, I think we should create a plugin olm and than call it there as we have now for the scorecard

Reply:
See https://github.com/operator-framework/operator-sdk/pull/3490#discussion_r459643492.

Reply:
My suggestion is the same as the comment. I just not think that the extra actions made by the "sdk plugins" would be called in the Init. Are not they called after the first plugin be executed? So, would not we have a plugin execution that in its postScaffold we call another one?